playkit/version.gradle

```diff
@@ -1,6 +1,6 @@
 
 // PlayKit Library Version
-ext.playkitVersion = 'dev'
+ext.playkitVersion = '0.1.4'
```

Comment:
This value should only be changed in a release branch -- it must stay as "dev" in develop.